src/Application/Service/RequestRememberUserPasswordRequest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ * This file is part of the User library.
+ *
+ * (c) Beñat Espiña <benatespina@gmail.com>
+ * (c) Gorka Laucirica <gorka.lauzirika@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace BenGor\User\Application\Service;
+
+/**
+ * Request remember user password request class.
+ *
+ * @author Beñat Espiña <benatespina@gmail.com>
+ * @author Gorka Laucirica <gorka.lauzirika@gmail.com>
+ */
+final class RequestRememberUserPasswordRequest
+{
+    /**
+     * The user id.
+     *
+     * @var string
+     */
+    private $id;
```

Comment:
This should be an email. I want to receive an email in the given address... The form in this use case will be a single input with an email field